recipes/ois/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "1.5":
+    url: "https://github.com/wgois/OIS/archive/v1.5.tar.gz"
+    sha256: "aa3e6b840b1149bb06835e04365957f4659c08b3e3095a48b289050f1637a174"
+patches:
+  "1.5":
+    - patch_file: "patches/001_fix.patch"
```

Comment:
would be nice if the file name was more descriptive

Reply:
How about `0001-cmake-don-t-user-CMAKE_SOURCE_DIR.patch`?